src/main/kotlin/no/nav/syfo/domain/Vurdering.kt

```diff
@@ -172,28 +209,44 @@ sealed interface Vurdering {
                     journalpostId = journalpostId,
                     publishedAt = publishedAt
                 )
+
+                VurderingType.IKKE_AKTUELL -> IkkeAktuell(
+                    uuid = uuid,
+                    createdAt = createdAt,
+                    personident = personident,
+                    veilederident = veilederident,
+                    arsak = VurderingArsak.valueOf(arsak!!),
+                    document = document,
+                    journalpostId = journalpostId,
+                    publishedAt = publishedAt
+                )
             }
         }
     }
 }
 
 enum class VurderingType(val isFinal: Boolean) {
-    FORHANDSVARSEL(false), OPPFYLT(true), AVSLAG(true);
+    FORHANDSVARSEL(false), OPPFYLT(true), AVSLAG(true), IKKE_AKTUELL(true);
 }
 
 fun VurderingType.getDokumentTittel(): String = when (this) {
     VurderingType.FORHANDSVARSEL -> "Forhåndsvarsel om avslag på sykepenger"
-    VurderingType.OPPFYLT -> "Vurdering av §8-4 arbeidsuførhet"
+    VurderingType.OPPFYLT, VurderingType.IKKE_AKTUELL -> "Vurdering av §8-4 arbeidsuførhet"
     VurderingType.AVSLAG -> "Innstilling om avslag"
 }
 
 fun VurderingType.getBrevkode(): BrevkodeType = when (this) {
     VurderingType.FORHANDSVARSEL -> BrevkodeType.ARBEIDSUFORHET_FORHANDSVARSEL
-    VurderingType.OPPFYLT -> BrevkodeType.ARBEIDSUFORHET_VURDERING
+    VurderingType.OPPFYLT, VurderingType.IKKE_AKTUELL -> BrevkodeType.ARBEIDSUFORHET_VURDERING
     VurderingType.AVSLAG -> BrevkodeType.ARBEIDSUFORHET_AVSLAG
 }
 
 fun VurderingType.getJournalpostType(): JournalpostType = when (this) {
-    VurderingType.FORHANDSVARSEL, VurderingType.OPPFYLT -> JournalpostType.UTGAAENDE
+    VurderingType.FORHANDSVARSEL, VurderingType.OPPFYLT, VurderingType.IKKE_AKTUELL -> JournalpostType.UTGAAENDE
```

Comment:
Journalfører som utgående dokument slik at blir synlig for bruker på innloggede sider.

Reply:
Så da viser vi også oppfylt, ja. Bra 👍🏼